### PR TITLE
Ordered postcode search

### PR DESCRIPTION
**Pull request by camilleldn**
linked to https://github.com/LandRegistry/digital-register-elasticsearch-updater/pull/21, do not merge until acceptance tests are done :-D

### CODE REVIEW

File: service/server.py [hunk 1]
```diff
@@ -108,6 +108,22 @@ def create_search(doc_type):
     return search
 
 
+def create_ordered_search(doc_type):
+    client = Elasticsearch([ELASTIC_SEARCH_ENDPOINT])
+    search = Search(using=client, index='landregistry', doc_type=doc_type)
+    search.sort(
+        'street_name',
```
Comment (lgonczar) [inline service/server.py]: Do the sorting results meet our expectations regarding the missing fields?

**Comment by lgonczar** [comment]
Reviewed. Looks good, left one question.

**Comment by camilleldn** [comment]
Please merge with associated pull requests on elasticsearch-updater and feeder :) 
merge order should be:
1) updater
2) feeder and api
thanks :heart:

**Comment by camilleldn** [comment]
Build broken due to Jenkins issues, tests actually passing :( Mikael is looking into it

File: service/server.py (hunk 1 above)
Comment (camilleldn) [inline service/server.py]: yup, see acceptance test results :)

**Comment by lgonczar** [comment]
Good to merge

**Comment by Pheonnexx** [comment]
LGTM

**Pull request merged**